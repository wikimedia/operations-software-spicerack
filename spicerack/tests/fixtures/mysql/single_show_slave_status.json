{
    "Connect_Retry": 60,
    "Exec_Master_Log_Pos": 123456782,
    "Gtid_IO_Pos": "123456780-123456781-123456782,123456784-123456785-123456786",
    "Last_Errno": 0,
    "Last_Error": "",
    "Last_IO_Errno": 0,
    "Last_IO_Error": "",
    "Last_SQL_Errno": 0,
    "Last_SQL_Error": "",
    "Master_Host": "host1.example.org",
    "Master_Log_File": "host1-bin.001234",
    "Master_Port": 3306,
    "Master_SSL_Allowed": "Yes",
    "Master_SSL_CA_File": "",
    "Master_SSL_CA_Path": "",
    "Master_SSL_Cert": "",
    "Master_SSL_Cipher": "",
    "Master_SSL_Crl": "",
    "Master_SSL_Crlpath": "",
    "Master_SSL_Key": "",
    "Master_SSL_Verify_Server_Cert": "No",
    "Master_Server_Id": 123456784,
    "Master_User": "user1",
    "Parallel_Mode": "optimistic",
    "Read_Master_Log_Pos": 123456780,
    "Relay_Log_File": "host1-relay-bin.000012",
    "Relay_Log_Pos": 123456781,
    "Relay_Log_Space": 123456783,
    "Relay_Master_Log_File": "host1-bin.001234",
    "Replicate_Do_DB": "",
    "Replicate_Do_Domain_Ids": "",
    "Replicate_Do_Table": "",
    "Replicate_Ignore_DB": "",
    "Replicate_Ignore_Domain_Ids": "",
    "Replicate_Ignore_Server_Ids": "",
    "Replicate_Ignore_Table": "",
    "Replicate_Wild_Do_Table": "",
    "Replicate_Wild_Ignore_Table": "",
    "SQL_Delay": 0,
    "SQL_Remaining_Delay": null,
    "Seconds_Behind_Master": 0,
    "Skip_Counter": 0,
    "Slave_DDL_Groups": 0,
    "Slave_IO_Running": "Yes",
    "Slave_IO_State": "Waiting for master to send event",
    "Slave_Non_Transactional_Groups": 0,
    "Slave_SQL_Running": "Yes",
    "Slave_SQL_Running_State": "Slave has read all relay log; waiting for more updates",
    "Slave_Transactional_Groups": 123456,
    "Until_Condition": "None",
    "Until_Log_File": "",
    "Until_Log_Pos": 0,
    "Using_Gtid": "Slave_Pos"
}
