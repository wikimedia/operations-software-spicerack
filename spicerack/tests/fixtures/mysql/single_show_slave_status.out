*************************** 1. row ***************************
                Slave_IO_State: Waiting for master to send event
                   Master_Host: host1.example.org
                   Master_User: user1
                   Master_Port: 3306
                 Connect_Retry: 60
               Master_Log_File: host1-bin.001234
           Read_Master_Log_Pos: 123456780
                Relay_Log_File: host1-relay-bin.000012
                 Relay_Log_Pos: 123456781
         Relay_Master_Log_File: host2-bin.001234
              Slave_IO_Running: Yes
             Slave_SQL_Running: Yes
               Replicate_Do_DB: 
           Replicate_Ignore_DB: 
            Replicate_Do_Table: 
        Replicate_Ignore_Table: 
       Replicate_Wild_Do_Table: 
   Replicate_Wild_Ignore_Table: 
                    Last_Errno: 0
                    Last_Error: 
                  Skip_Counter: 0
           Exec_Master_Log_Pos: 123456782
               Relay_Log_Space: 123456783
               Until_Condition: None
                Until_Log_File: 
                 Until_Log_Pos: 0
            Master_SSL_Allowed: Yes
            Master_SSL_CA_File: 
            Master_SSL_CA_Path: 
               Master_SSL_Cert: 
             Master_SSL_Cipher: 
                Master_SSL_Key: 
         Seconds_Behind_Master: 0
 Master_SSL_Verify_Server_Cert: No
                 Last_IO_Errno: 0
                 Last_IO_Error: 
                Last_SQL_Errno: 0
                Last_SQL_Error: 
   Replicate_Ignore_Server_Ids: 
              Master_Server_Id: 123456784
                Master_SSL_Crl: 
            Master_SSL_Crlpath: 
                    Using_Gtid: Slave_Pos
                   Gtid_IO_Pos: 180355171-180355171-148310907,180359172-180359172-49702203,171970637-171970637-2116621969,171978826-171978826-1025218266,180363268-180363268-3447080256,171974720-171974720-2572451842,171970745-171970745-3901487599,171978774-171978774-5,180355190-180355190-1378262411,171970572-171970572-3935877275,171970661-171970661-3655324752,0-171970637-5484646134
       Replicate_Do_Domain_Ids: 
   Replicate_Ignore_Domain_Ids: 
                 Parallel_Mode: optimistic
                     SQL_Delay: 0
           SQL_Remaining_Delay: NULL
       Slave_SQL_Running_State: Slave has read all relay log; waiting for more updates
              Slave_DDL_Groups: 2
Slave_Non_Transactional_Groups: 0
    Slave_Transactional_Groups: 232276084
